draft-ietf-webpush-protocol.xml

```diff
@@ -1167,19 +1167,10 @@ HEADERS      [stream 82] +END_STREAM
           medium for key agreement.
         </t>
         <t>
-          <!-- Scrub references to specific API(s) -->
-          The Web Push API codifies this practice by requiring that each push
-          subscription created by the browser be bound to a browser generated
-          encryption key.  Pushed messages are authenticated and decrypted by
-          the browser before delivery to applications.  This scheme ensures that
-          the push service is unable to examine the contents of push messages.
-        </t>
-        <t>
-          The public key for a subscription ensures that applications using that
-          subscription can identify messages from unknown sources and discard
-          them.  This depends on the public key only being disclosed to entities
-          that are authorized to send messages on the channel.  The push service
-          does not require access to this public key.
+          For its requirements, the <xref target="API">W3C Push API</xref> has adopted 
+          <xref target="I-D.ietf-webpush-encryption">Message Encryption for WebPush</xref>
+          to secure the content of messages from the push service. Other scenarios can 
+          be addressed by similar policies.

```

Comment:
I think of it more like a central place to place the MUST rather than having repetitive language in the subsections. Do you have a specific change in mind?